libs/s25main/TerrainRenderer.cpp

```diff
@@ -44,6 +45,15 @@
  * providing an index and a count into the above arrays.
  */
 
+namespace {
+// gl4es has some problems with GL_MODULATE so we need to change the colors without using a gl function
+#if RTTR_OGL_GL4ES
+constexpr float TEXTURE_COLOR_DIVISOR = 1;
+#else
+constexpr float TEXTURE_COLOR_DIVISOR = 2;
```

Comment:
Can you use this constant in `glTexEnvf(GL_TEXTURE_ENV, GL_RGB_SCALE, 2.0f);` so one can tell where the 2 comes from/is used?